components/prism-arturo.js

```diff
@@ -0,0 +1,49 @@
+Prism.languages.arturo = {
+	'comment': /;.*/,
+
+	'character': {
+		pattern: /`.`/,
+		alias: 'char'
+	},
+
+	'string': {
+		pattern: /"(?:[^"\\]|\\.)*"/,
```

Comment:
This currently allows strings like
```
"foo
bar"
```
which seems to be incorrect.

Did you mean `/"(?:[^"\\\r\n]|\\.)*"/`?

Reply:
On the topic of strings. How about supporting the other kinds of strings as well?

```js
'string': {
  pattern: /"..."|».*|\{:[\s\S]*?:\}|\{[\s\S]*?\}|^-{6}$[\s\S]*/m,
  greedy: true
},
```

Reply:
You are right. I fixed it.

Regarding curly strings, etc I had thought of leaving it for some other day, but I guess your suggestion works beautifully. Thanks! 😄